torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -350,6 +352,17 @@ RegisterOperators reg({
             return 0;
           };
         }),
+    Operator(
+        "prim::SumToSize(Tensor(a) self, int[] size) -> Tensor(a)",
+        [](const Node* node) {
+          return [=](Stack& stack) {
+            at::Tensor self;
+            Shared<IntList> desired_sizes;
```

Comment:
What is `Shared<...>`?

Reply:
It's some kind of shared pointer that's implemented in this file. All list functions use code like this (search for e.g. `listNe`)